src/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nativescript-fresco",
-  "version": "3.1.0",
+  "version": "4.0.0",
```

Comment:
Major version is expected to have breaking changes. If it does, they should be described in a migration guide in the plugin's readme (see imagepicker for example)

Reply:
Added breaking changes to PR and readme